wordpress-mobile/WordPress-iOS
# Make sure post has the latest content before uploading.

Fixes #5923

Found a race condition between the images source being saved to the post, and the possibility of the user save the post. If tapped to quickly sometimes the post was upload to server with the content before  the images sources was set.

To test:
 - Create a new post
 - Upload some images
 - The moment they show uploaded on the editor tap on Post/Save
 - Open the post again
 - Check the image source is correct.

Needs review: @aerych
